docs/source/_static/docversions.js

```diff
@@ -0,0 +1,61 @@
+/* Create version-related dynamic output: instert_version_links() will output
```

Comment:
```suggestion
/* Create version-related dynamic output: insert_version_links() will output
```

Just a nit, but there's a typo here